arduino/arduino_config_msg config_msg
---
int8 ret